src/components/FeatureSummary.module.scss

```diff
@@ -1,10 +1,27 @@
+@import "../styles/variables";
+
+$font-size: 1.5rem;
+
 .feature-summary {
   display: flex;
   flex-direction: column;
   justify-content: center;
+  align-items: center;
   margin: 0 2rem;
+  height: 300px;
+
+  > p:nth-child(1) {
```

Comment:
Just as a warning, you are targetting child elements without specifying the type, you could be needing "nth-of-type" instead.